daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/ledgerinteraction/IdeLedgerClient.scala

```diff
@@ -47,7 +47,11 @@ class IdeLedgerClient(
   def currentSubmission: Option[ScenarioRunner.CurrentSubmission] = _currentSubmission
 
   private[this] val preprocessor =
-    new preprocessing.CommandPreprocessor(compiledPackages.interface, requiredCidSuffix = false)
+    new preprocessing.CommandPreprocessor(
+      compiledPackages.interface,
+      requireV1ContractId = false,
```

Comment:
Can we not enforce v1 cids in Daml Studio?

Reply:
will try.